Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hashlink"
-version = "0.9.1"
+version = "0.10.0"
```

Comment:
Not entirely sure if hashbrown is intended to be a public dependency for hashlink, but given that the default hasher parameter at least exposes the `DefaultHashBuilder`, I think it should probably count as public?

Reply:
You're completely right that it does, but it *shouldn't*, so the version bump is necessary here. In the next version we should guard that so we don't have to bump the public version again... Is there anything else you can see left that's re-exported or used as a default parameter? That's the only thing left, right?

Reply:
Thank you!